tools/arweave/check-failed-transactions.js

```diff
@@ -0,0 +1,54 @@
+const axios = require("axios");
+const Arweave = require("arweave/node");
+
+// const LIMIT = 300;
+const SKIP = 5;
+const LIMIT = 24 * 60;
```

Comment:
What does the limit stand for - is it the number of recent transactions?

Reply:
yes, it's the number of recent transactions